### ISSUE TITLE
Individually update videos.json and Aerial Screensaver App

### ISSUE THREAD

**Issue by VasilisPat** [OrangeJedi/Aerial#41]
**All this would apply after the original auto-update is introduced.**

Since the videos list is updated frequently by Apple and there is no way to keep up releasing new version of the app to accomodate the new videos every time this happens, is there any chance that the videos.json file is updated separately from the main app. One task would only check for new videos and the other only for new app versions, assuming that an app update isn't required so that the new videos can be added to the settings list.

**Comment by OrangeJedi** [comment]
Yes, I've been thinking about this for a while, and think it can be done, once we have more 'auto-update' features.

**Comment by VasilisPat** [comment]
Just a question to see if I can help, are you planning to use the electron included "auto-update" package?

**Comment by OrangeJedi** [comment]
Maybe. I haven't really looked into it, but it does look promising.

**Comment by Alexyoe** [comment]
How are we getting the videos.json currently?

**Comment by OrangeJedi** [comment]
It's a messy process. All the video info comes from Apple at this [link](https://sylvan.apple.com/Aerials/resources-15.tar). The problem with auto updating is that the URL changes whenever Apple releases new videos (which only happens once a year) and that the format of the file has changed to. You can see more of what I wrote [here](https://github.com/OrangeJedi/Aerial/issues/91#issuecomment-1269227249). Updating the other video information, like points of interest, is we even worse as the files are compressed in a wild format. Currently I use a few custom scripts and a lot of manuel formating to add them to the video file.

If the update notifyer in v1.1.0 works well, I'm planning on having Aerial check for new versions of the video.js file on the master branch and download the new file if there is a chance. Currently I thinks that's the best bet for an auto updating video file.
